### PR TITLE
Reuse Settings General page to be Site Tools

### PR DESCRIPTION
**Pull request by candy02058912**
Fixes https://github.com/Automattic/dotcom-forge/issues/5705

## Proposed Changes

* Use `wp-admin` interface to choose to render Site Tools (wp-admin settings removed) or General Settings
* Removes footer credit option (classic view only)
* Renames "Site Tools" / "Tools" => "Settings"
* Moves "Settings" to the bottom of the menu list
* Removes site visibility option as this is now available in wp-admin. https://github.com/Automattic/wp-calypso/issues/87210

## Testing Instructions

Classic view + flag enabled:

* Go to Global site view and click on Settings
* Reduced general settings should be shown

Default view + flag or no flag:

* Clicking "Settings > General" should load general settings unchanged

Before /settings/site-tools/:site

Screenshot 2024-02-21 at 23-24-32 Site Tools ‹ Creator Test Site - Atomic — WordPress com

After /settings/general/:site

Screenshot 2024-02-21 at 23-23-54 Settings ‹ Creator Test Site - Atomic — WordPress com

### CODE REVIEW

File: client/my-sites/site-settings/main.jsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ import './style.scss';
 
 const getTitle = ( isAdminInterface ) => {
 	if ( isAdminInterface ) {
-		return translate( 'Site Tools' );
+		return translate( 'Settings' );
 	}
```
Comment (miksansegundo) [inline client/my-sites/site-settings/main.jsx:25]: Much better 👍

**Review by miksansegundo — APPROVED** [review]
LGTM 👍 

Thanks!

**Review by lsl — DISMISSED** [review]
Has bugs (working on it)

File: client/my-sites/site-settings/form-general.jsx [hunk 2]
```diff
@@ -345,191 +343,6 @@ export class SiteSettingsFormGeneral extends Component {
 		);
 	}
 
-	visibilityOptionsComingSoon() {
```
Comment (lsl) [inline client/my-sites/site-settings/form-general.jsx:348]: dead code 6c686be544f948569a4afd78ee20a4b417a14271

File: client/my-sites/site-settings/main.jsx [hunk 3]
```diff
@@ -17,31 +18,44 @@ import GeneralSettings from './section-general';
...
+const getTitle = ( isClassicView ) => {
+	if ( isClassicView ) {
+		return translate( 'Settings' );
+	}
+	return translate( 'General Settings' );
+};
+
+const getSubtitle = ( isClassicView ) => {
+	if ( isClassicView ) {
+		return translate( 'Manage your site settings, including site visibility, and more.' );
```
Comment (lsl) [inline client/my-sites/site-settings/main.jsx:30]: Copy could use work after removing the actual "site visibility" option from here fee4f0366849efd2554851c43b081103081f87d9

**Review by DustyReagan — APPROVED** [review]
Tested and looks good!

File: client/my-sites/site-settings/main.jsx [hunk 4]
```diff
@@ -17,31 +19,46 @@ import GeneralSettings from './section-general';
 
 import './style.scss';
 
+const getTitle = ( isClassicView ) => {
+	if ( isClassicView ) {
+		return translate( 'Settings' );
+	}
```
Comment (DustyReagan) [inline client/my-sites/site-settings/main.jsx]: Should we check for `layout/dotcom-nav-redesign` here too?

Comment (DustyReagan) [reply client/my-sites/site-settings/main.jsx]: I went ahead and added it. It's easy enough to remove if needed.

**Pull request merged**